src/features/collaborator-avatars/CollaboratorList.scss

```diff
@@ -2,7 +2,6 @@
 
 .usm-collaborator-list {
     .collaborator-list {
-        min-width: 400px;
```

Comment:
when `is-responsive-web` is enabled, the `min-width` on this class is `0` with the intention of resetting the `400px` style that is set here. so logically neither of them are needed anymore.